sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetPartitionReaderFactory.scala

```diff
@@ -134,10 +134,14 @@ case class ParquetPartitionReaderFactory(
         private var hasNext = true
         private lazy val row: InternalRow = {
           val footer = getFooter(file)
+
+          val partitionValues = AggregatePushDownUtils.reOrderPartitionCol(
```

Comment:
can we call this in `ParquetUtils.createAggInternalRowFromFooter`? it seems we always call `AggregatePushDownUtils.reOrderPartitionCol` then followed by `ParquetUtils.createAggInternalRowFromFooter`.